lib/ES/Repo.pm

```diff
@@ -316,6 +341,7 @@ sub checkout_to {
 #===================================
     my ( $self, $destination ) = @_;
 
+    die 'sub_dir not supported with checkout_to' if %{ $self->{sub_dirs}};
```

Comment:
This method is only used by the `target_repo` so we don't need to support `sub_dirs`. In fact, it doesn't make sense to have it!